skore/src/skore/ui/project_routes.py

```diff
@@ -55,7 +55,7 @@ def __serialize_project(project: Project) -> SerializedProject:
                 value = item.array.tolist()
                 media_type = "text/markdown"
             elif isinstance(item, PandasDataFrameItem):
-                value = item.dataframe.to_dict(orient="tight")
+                value = item.dataframe.fillna("missing value").to_dict(orient="tight")
```

Comment:
This should be also applied at least to numpy, polars dataframe, polars/pandas series.

Reply:
As polars API is less permissive I chose to convert to pandas (which happens anyway) then to fill the holes.